File: add/codesnippet/CSharp/73a8b6eb-26fe-4a75-8642-_1.cs
            // Tests for the presence of a CodeTypeReference in the 
            // collection, and retrieves its index if it is found.
            CodeTypeReference testReference = new CodeTypeReference(typeof(bool));
            int itemIndex = -1;
            if( collection.Contains( testReference ) )
                itemIndex = collection.IndexOf( testReference );

File: add/codesnippet/VisualBasic/73a8b6eb-26fe-4a75-8642-_1.vb
        ' Tests for the presence of a CodeTypeReference in the 
        ' collection, and retrieves its index if it is found.
        Dim testReference As New CodeTypeReference(GetType(Boolean))
        Dim itemIndex As Integer = -1
        If collection.Contains(testReference) Then
            itemIndex = collection.IndexOf(testReference)
        End If